src/validate.js

```diff
@@ -1730,7 +1731,8 @@ function F_VALIDATE(disabledHL) {
 		}
 
 		// emulate WMECH_color
-		var segmentGeometry = document.getElementById(rawSegment.geometry.id);
+		var geoID = rawSegment.getOLGeometry().id;
```

Comment:
I'm not sure how `geoID` is used?

Reply:
That was the new way to get the geometry ID, but it turns out the new getFeatureElementByDataModel does not use that, so I will remove that line.